kafka/consumer.go

```diff
@@ -907,3 +907,13 @@ func (c *Consumer) handleRebalanceEvent(channel chan Event, rkev *C.rd_kafka_eve
 
 	return nil
 }
+
+// SetSaslCredentials sets the SASL credentials used for this consumer. The new credentials
+// will overwrite the old ones (which were set when creating the consumer or by a previous
+// call to SetSaslCredentials). The new credentials will be used the next time the
+// consumer needs to establish a connection to the broker. This method will *not*
```

Comment:
I think "needs to authenticate to brokers" is more correct, and will also cover the future case of reauthentication

Reply:
Both done